qiskit/transpiler/passes/layout/apply_layout.py

```diff
@@ -44,7 +44,7 @@ def run(self, dag):
             raise TranspilerError(
                 "No 'layout' is found in property_set. Please run a Layout pass in advance."
             )
-        if len(layout) != (1 + max(layout.get_physical_bits())):
+        if len(layout) != (1 + max(layout._p2v)):
```

Comment:
```suggestion
        if len(layout) != (1 + max(layout.get_physical_bits())):
```
If this is only one call, outside a loop, can we use the public API here?

Reply:
Sure, I doubt we'll see the difference here